Packs/PICUS/Integrations/PICUS/PICUS.yml

```
@@ -0,0 +1,220 @@
+  type: 4
+- defaultvalue: "false"
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: "false"
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Continuous Breach And Attack Simulation
+display: PICUS
+name: PICUS
+script:
+  commands:
+  - arguments:
+    - description: Result begin date
+      name: begin_date
+      required: true
+    - description: Result end date
+      name: end_date
+      required: true
+    - description: Victem Peer
+      name: trusted
+      required: true
+    - description: Attacker Peer
+      name: untrusted
+      required: true
+    description: 'Attack Result - Makes a comparison of the given vector''s results.
+      Example Command: !picus-vector-compare begin_date=2020-01-20 end_date=2021-01-20
+      trusted=Trusted_Peer1 untrusted=Untrusted_Peer1'
+    name: picus-vector-compare
+  - arguments:
+    - auto: PREDEFINED
+      defaultValue: insecure
+      description: Attack results that should be filtered. Secure or Insecure
+      name: attack_result
+      predefined:
+      - secure
+      - insecure
+      required: true
+    - description: Threat release date filter start of the date range
+      name: begin_date
+      required: true
+    - defaultValue: "False"
+      description: 'Default: false -  Process Results of Scenario Details have console
+        output information which can be in large sizes so this data is disabled by
+        default'
+      name: console_output_info
+    - description: "\t string Default: \"Today's date formatted YYYY-mm-dd\" Threat\
+        \ release date filter end of the date range if a begin date is given and end\
+        \ date not, default will be used"
+      name: end_date
+      required: true
+    - defaultValue: "null"
+      description: 'Default: "null" allowed time formats RFC822, RFC822Z, RFC1123,
+        RFC1123Z, RFC850, RFC3339'
+      name: from_time
+    - defaultValue: "1"
+      description: 'Default: 1 Requested page number'
+      name: page
+    - default: true
+      defaultValue: "50"
+      description: 'Default: 50 Requested data size'
+      name: size
+    - description: '"threat_parameters": { "begin_date": "2018-10-29", "categories":
+        [ [ "Malicious Code" ], [ "Attack Scenario", "Defense Evasion", "Indicator
+        Removal from Tools" ] ],'
+      name: threat_parameters
+    - description: ' Array of objects (PeerPairParams) Vectors.(Trusted Peer)'
+      name: vector1
+      required: true
+    - description: ' Array of objects (PeerPairParams) Vectors.(Untrusted Peer)'
+      name: vector2
+      required: true
+    description: Returns the list of the attack results\nhave optional parameters
+      for pagination and filtration. \nExample Command:\n !picus-attack-result-list
+      attack_result=insecure begin_date=2020-01-01 end_date=2020-09-05  vector1=Trusted-Peer1
+      vector2=Untrusted-Peer1
```

Please fix it in all places.

```suggestion
    description: Returns the list of the attack results. Have optional parameters
      for pagination and filtration. Example Command: !picus-attack-result-list
      attack_result=insecure begin_date=2020-01-01 end_date=2020-09-05  vector1=Trusted-Peer1
      vector2=Untrusted-Peer1
```